www/src/pages/foundations/elevation.jsx

```diff
@@ -0,0 +1,280 @@
+import React, { useState, useEffect } from 'react';
+import PropTypes from 'prop-types';
+import { CopyToClipboard } from 'react-copy-to-clipboard';
+import SEO from '../../components/SEO';
+import Layout from '../../components/PageLayout';
+import {
+  Button, Form, Container, Input, Toast,
+} from '~paragon-react'; // eslint-disable-line import/no-unresolved
+
+const boxShadowSides = ['down', 'up', 'right', 'left', 'centered'];
+const boxShadowLevels = [1, 2, 3, 4, 5];
+
+const controlsProps = [
+  { key: 'x', name: 'Offset X' },
+  { key: 'y', name: 'Offset Y' },
+  { key: 'blur', name: 'Blur' },
+  { key: 'spread', name: 'Spread' },
+  { key: 'color', name: 'Color' },
+];
+
+const BoxShadowNode = () => {
+  const [showToast, setShowToast] = useState(false);
+  const [isCopied, setIsCopied] = useState(false);
+
+  const handleBoxShadowCopy = () => {
+    setIsCopied(true);
+    setShowToast(true);
+  };
+
+  const boxShadowCells = boxShadowLevels.map(level => (
+    boxShadowSides.map(side => (
+      <CopyToClipboard
+        key={side}
+        text={`@include pgn-box-shadow(${level}, "${side}");`}
+        onCopy={handleBoxShadowCopy}
+      >
+        <div
+          className={`pgn-doc__box-shadow-cell box-shadow-${side}-${level}`}
+          /* eslint-disable-next-line jsx-a11y/no-noninteractive-tabindex */
+          tabIndex={0}
+          aria-label="Box-shadow cell"
+        />
+      </CopyToClipboard>
+    ))
+  ));
+
+  return (
+    <div className="pgn-doc__box-shadow-cells">
+      { boxShadowCells }
+      {isCopied && (
```

Comment:
I don't think you need `isCopied` variable in this component, you can just use `Toast`'s `show` prop to decided whether it needs to be rendered, seems unnecessary to use another variable for this.

Reply:
Done